torch/quantization/ns/numeric_suite_core_apis_fx.py

```diff
@@ -28,11 +27,69 @@
     create_a_shadows_b,
 )
 
+from .ns_types import (
+    NSSingleResultValuesType,
+)
+
 from typing import Dict, Tuple, Callable, List, Any
 
-class NSSingleResultValuesType(str, enum.Enum):
-    WEIGHT = 'weight'
-    NODE_OUTPUT = 'node_output'
+
+class OutputLogger(nn.Module):
```

Comment:
Maybe we can think of another name for this logger later, since we also use it to log input and current name might be a little confusing.